utils/python/CIME/XML/pio.py

```diff
@@ -17,13 +17,21 @@ def __init__(self, infile=None, files=None):
 
         EntryID.__init__(self, infile)
 
-    def get_defaults(self, grid=None, compset=None, mach=None, compiler=None):
+    def get_defaults(self, grid=None, compset=None, mach=None, compiler=None, mpilib=None):
         # should we have a env_pio file
         defaults = {}
+
+        # Load args into attribute dict
+        attributes = {}
+        for attrib in ["grid", "compset", "mach", "compiler", "mpilib"]:
```

Comment:
Bug fix: need to look at mpilib in order to pick up netcdf instead of pnetcdf if mpilib is mpi-serial.